proto/transform.proto

```diff
@@ -12,7 +12,6 @@ message TransformQuery {
 
 message TransformRequest {
   TimeRange timeRange = 1;
-  DatasourceInfo datasource = 2;
   repeated TransformQuery queries = 3;
```

Comment:
Here's the reason it still worked in Grafana. `queries` still has number 3, should be 2 and requestId 3.